### ISSUE TITLE
EDDI doesn't read out multi crew text messages

### ISSUE THREAD

**Issue by TalShaf** [EDCD/EDDI#291]
## EDDI version in which issue found
[2.4.6-b1]

## Steps to reproduce

Receive message from multi crew member.

## Expected
EDDI should read out the message from a Commander

## Observed
EDDI reads out "message received from NPC. Message reads", but doesn't read out the message itself.

## Investigation
Journal entries detailing some of the messages received:

```
{ "timestamp":"2017-12-06T22:40:54Z", "event":"ReceiveText", "From":"Nexonoid", "Message":"whats up" }
{ "timestamp":"2017-12-06T22:41:02Z", "event":"Music", "MusicTrack":"SystemMap" }
{ "timestamp":"2017-12-06T22:41:03Z", "event":"ReceiveText", "From":"Nexonoid", "Message":"has the space madness kicked in yet? :P " }
{ "timestamp":"2017-12-06T22:41:10Z", "event":"Music", "MusicTrack":"Supercruise" }
{ "timestamp":"2017-12-06T22:41:40Z", "event":"SendText", "To":"wing", "Message":"Not yet. Havent been out that long." }
{ "timestamp":"2017-12-06T22:42:01Z", "event":"StartJump", "JumpType":"Hyperspace", "StarSystem":"Blae Hype UA-D c3", "StarClass":"K" }
{ "timestamp":"2017-12-06T22:42:06Z", "event":"Music", "MusicTrack":"NoTrack" }

```

**Comment by richardbuckle** [comment]
So the ReceiveText event is missing its channel parameter when it's from a multi-crew member.

**Comment by Tkael** [comment]
Yep. And defaulting to "NPC"

**Comment by Tkael** [comment]
I've created a PR for this but I'll defer to VB since he claimed it 1st. :-)

**Comment by Tkael** [comment]
Closed pending acceptance of PR #292.
